app/controllers/answers.js

```diff
@@ -1,55 +1,69 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    Answer = mongoose.model('Answer'),
-    _ = require('underscore');
+const mongoose = require('mongoose');
+const async = require('async');
+const Answer = require('../models/answer');
+const _ = require('underscore');
 
 
 /**
- * Find answer by id
+ * [answer description]
+ * @param  {Object}   req  [description]
+ * @param  {Object}   res  [description]
+ * @param  {Function} next [description]
+ * @param  {Integer}   id   [description]
+ * @return {undefined}        calls the next function in the middlewae chain
```

Comment:
Your return type in not correct, it can either by void or null or mixed.

Reply:
Okay. Thanks.
I'll fixed that and the others as well.

I paid no mind the accuracy because I was just using those changes to test that my changes were actually triggering a build